builtin/credential/cert/path_certs.go

```diff
@@ -90,6 +91,11 @@ from the AuthorityInformationAccess extension on the certificate being inspected
 				Default:     false,
 				Description: "If set to true, rather than accepting the first successful OCSP response, query all servers and consider the certificate valid only if all servers agree.",
 			},
+			"ocsp_this_update_max_ttl": {
```

Comment:
I'd call it max_age actually.  TTL doesn't feel quite like what now() - this_update means.

Reply:
Renamed within https://github.com/hashicorp/vault/pull/25912/commits/929d092d19d1d7bec21639c2f1ead2815d117881